retry/retry.go

```diff
@@ -153,9 +146,30 @@ func (r *retrier) attempt(continueFn ContinueFn, fn Fn) error {
 			return err
 		}
 		r.metrics.errors.Inc(1)
-		curr = int64(float64(curr) * r.backoffFactor)
 	}
 	r.metrics.errorsFinal.Inc(1)
 
 	return err
 }
+
+// BackoffNanos calculates the backoff for a retry in nanoseconds.
+func BackoffNanos(
```

Comment:
should this be private?

Reply:
I need it to be public to be reused.

Reply:
lol, yea I didn't read the PR message :)